Makefile

```diff
@@ -1,12 +1,15 @@
-.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push
+.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push container-frontend-auth container-frontend-auth-push container-frontend-noauth container-frontend-noauth-push
 
 CONTAINER_TAG ?= tsidentity/tornjak-spire-server:latest
+CONTAINER_TAG_FRONTEND ?= tsidentity/tornjak-ui:latest
```

Comment:
I think we should stay consistent and use 'be' for Back-end and 'fe' for Front-end'. In this case the image should be `tornajk-fe`

Reply:
done